### DISCUSSION TITLE
Customization of status icons

### DISCUSSION THREAD

**Discussion by quenthal** [Blackymas/NSPanel_HA_Blueprint#1081] [Ideas]
I'm using relay 1 for thermostat and relay 2 is not connected.

When thermostat is turned on, both "1" and heating status icons turn on.  If it was possible, I'd rather either arrange those freely, or get rid of relays' icons completely. However, this is reallly minor thing, but anyway :)

**Comment by edwardtfn** [comment]
I haven't tested, but I believe if you select in you blueprint setting the icon `mdi:blank` for that relay, it will never show again. In fact the panel will be plotting the icon, but it won't be visible. 😉

**Comment by edwardtfn** [comment]
- #1081
